problems/my_zip.rb

```diff
@@ -1,9 +1,12 @@
 class Array
-  # ex.
-  # [1,2,3].my_zip([4,5,6], [7,8,9]) 
+  # Define a method `my_zip(*arrays)` that merges 
```

Comment:
Should be `Array#my_zip` or otherwise specify that it's monkey-patched onto the Array class